lefthook.yml

```diff
@@ -0,0 +1,43 @@
+skip_output:
+  - meta
+  - execution_out
+  - skips
+
+min_version: 1.4.3
+
+rc: ./lefthook.rc
+
+pre-commit:
+  parallel: true
+  commands:
+    frontend-betterer:
+      glob: '*.{ts,tsx}'
+      run: yarn betterer precommit {staged_files}
+      stage_fixed: true
+
+    frontend-eslint:
```

Comment:
given this runs prettier as well, could we rename it to not just be `eslint` specific? `frontend-lint` or `frontend-lint-and-format` or something? 🤔